### PR TITLE
Disable zip download for azure devops download task

### PR DESCRIPTION
**Pull request by safern**
Workaround suggested by azure devops for: https://github.com/dotnet/runtime/issues/32805

### CODE REVIEW

File: eng/pipelines/common/xplat-setup.yml [hunk 1]
```diff
@@ -12,6 +12,11 @@ jobs:
 - template: ${{ coalesce(parameters.helixQueuesTemplate, parameters.jobTemplate) }}
   parameters:
     variables:
+      # Workaround for azure devops flakyness when dowloading artifacts
```
Comment (trylek) [inline eng/pipelines/common/xplat-setup.yml]: Nit - typo, "flakiness"

**Review by trylek — APPROVED** [review]
LGTM, thank you!

**Review by jaredpar — APPROVED** [review]

File: eng/pipelines/common/xplat-setup.yml (hunk 1 above)
Comment (safern) [reply eng/pipelines/common/xplat-setup.yml]: Thanks. Will update.

**Comment by ericstj** [comment]
For the curious: https://github.com/microsoft/azure-pipelines-tasks/blob/0c7745117fba2dc18f3b1b341341159d07ecb819/Tasks/DownloadBuildArtifactsV0/main.ts#L259

**Review by ericstj — APPROVED** [review]

**Comment by safern** [comment]
Thanks @ericstj -- should we name it as `SYSTEM...`? It might be case sensitive in non-windows, right?

**Comment by jaredpar** [comment]
@safern 

Seems easy enough to check. When not using zip download the following will be in the log file:

> console.log(tl.loc("DownloadingContainerResource", artifact.resource.data));

**Comment by jaredpar** [comment]
@safern 

The pipelines code base uses the same variable name with different case in many places. Think it's safe to bet it's case insensitive.

**Comment by ericstj** [comment]
🤷‍♂ Not sure.  I don't have much experience with this codebase.  https://github.com/Microsoft/azure-pipelines-task-lib/blob/master/node/docs/azure-pipelines-task-lib.md#taskgetvariable- doesn't mention case, but I found https://github.com/microsoft/azure-pipelines-task-lib/blob/554c7e64203dbd29da9403ec3a153bab74ea2386/node/internal.ts#L259 which appears to normalize to upper during this call.

**Comment by jaredpar** [comment]
https://github.com/microsoft/azure-pipelines-tasks/search?q=getvariable+teamprojectid&unscoped_q=getvariable+teamprojectid

**Comment by safern** [comment]
Seems safe then. I'll leave it as is. 

> Seems easy enough to check. When not using zip download the following will be in the log file:
> console.log(tl.loc("DownloadingContainerResource", artifact.resource.data));

Also it seems like it worked, the localized resource in en-us is:
` "loc.messages.DownloadingContainerResource": "Downloading items from container resource %s",`

I see that in the build logs.

**Comment by jaredpar** [comment]
Windows_NT x86 release failure is #34010

**Comment by jaredpar** [comment]
OS failure is #34045

**Comment by jaredpar** [comment]
Mono tests aren't a know issue, nor does searching turn up any failures.

**Comment by trylek** [comment]
Well, it claims to be a long-running test, no surprise it timed out ;-).

**Comment by safern** [comment]
I'll merge, we can open an issue with runfo ?

**Comment by safern** [comment]
@davidsh @scalablecory maybe the timeout for the test was a network related thing? 
https://helix.dot.net/api/2019-06-17/jobs/33d09b8e-9faf-4c19-82bc-e7f7bdeefee7/workitems/System.Net.NetworkInformation.Functional.Tests/console

**Comment by jaredpar** [comment]
> I'll merge, we can open an issue with runfo ?

I checked that before my comment. Only one other failure in last 100 runs and it was another PR. Keeping an eye on that test now though.

**Comment by davidsh** [comment]
>@davidsh @scalablecory maybe the timeout for the test was a network related thing?

Not sure if we have this particular test hang before.  @wfurt ?

>  Discovering: System.Net.NetworkInformation.Functional.Tests (method display = ClassAndMethod, method display options = None)
  Discovered:  System.Net.NetworkInformation.Functional.Tests (found 75 of 92 test cases)
  Starting:    System.Net.NetworkInformation.Functional.Tests (parallel test collections = on, max threads = 2)
   System.Net.NetworkInformation.Functional.Tests: [Long Running Test] 'System.Net.NetworkInformation.Tests.IPInterfacePropertiesTest_Linux.IPInfoTest_IPv4Loopback_ProperAddress', Elapsed: 00:02:08
   System.Net.NetworkInformation.Functional.Tests: [Long Running Test] 'System.Net.NetworkInformation.Tests.IPInterfacePropertiesTest_Linux.IPInfoTest_IPv4Loopback_ProperAddress', Elapsed: 00:04:08
   System.Net.NetworkInformation.Functional.Tests: [Long Running Test] 'System.Net.NetworkInformation.Tests.IPInterfacePropertiesTest_Linux.IPInfoTest_IPv4Loopback_ProperAddress', Elapsed: 00:06:08
   System.Net.NetworkInformation.Functional.Tests: [Long Running Test] 'System.Net.NetworkInformation.Tests.IPInterfacePropertiesTest_Linux.IPInfoTest_IPv4Loopback_ProperAddress', Elapsed: 00:08:08
   System.Net.NetworkInformation.Functional.Tests: [Long Running Test] 'System.Net.NetworkInformation.Tests.IPInterfacePropertiesTest_Linux.IPInfoTest_IPv4Loopback_ProperAddress', Elapsed: 00:10:08
   System.Net.NetworkInformation.Functional.Tests: [Long Running Test] 'System.Net.NetworkInformation.Tests.IPInterfacePropertiesTest_Linux.IPInfoTest_IPv4Loopback_ProperAddress', Elapsed: 00:12:08
   System.Net.NetworkInformation.Functional.Tests: [Long Running Test] 'System.Net.NetworkInformation.Tests.IPInterfacePropertiesTest_Linux.IPInfoTest_IPv4Loopback_ProperAddress', Elapsed: 00:14:08

**Comment by wfurt** [comment]
no, I have not seen this one @davidsh but the the failure was on Mono runtime. (and we do not have that much experience with it yet)
The test is pretty trivial (GetAllNetworkInterfaces) but online most other tests it does use LINQ.

**Pull request merged**